Package.swift

```diff
@@ -31,7 +31,7 @@ let products: [Product] = [
 let dependencies: [Package.Dependency] = [
   .package(
     url: "https://github.com/grpc/grpc-swift.git",
-    exact: "2.0.0-beta.2"
+    branch: "main"
```

Comment:
@glbrntt note this change. I'm happy to take a different approach than this if you'd rather.

Reply:
Nope, this is good. We only switched to `exact` for the tagged releases, `main` is fine between releases.